extensions/indexes/spatial/pom.xml

```diff
@@ -46,7 +46,7 @@
   </scm>
 
     <properties>
-        <geotools.version>2.4.5</geotools.version>
+        <geotools.version>29.6</geotools.version>
```

Comment:
Nice.... did not expect such a version bump works; I tried some works on it recently, and some reading....